app/kubernetes/views/resource-pools/edit/resourcePool.html

```diff
@@ -396,13 +405,14 @@
                 <!-- !summary -->
 
                 <!-- actions -->
-                <div ng-if="ctrl.isAdmin && ctrl.isEditable" class="col-sm-12 form-section-title">
+                <div ng-if="ctrl.isAdmin && ctrl.pool.Namespace.Name !== 'default'" class="col-sm-12 form-section-title">
```

Comment:
use `namespaceHelper` for this?
when loading rp check if it's default and expose this state.